rusk/src/lib/http/chain.rs

```diff
@@ -124,4 +140,72 @@ impl RuskNode {
 
         Ok(ResponseData::new(serde_json::to_value(&info)?))
     }
+
+    /// Calculates various statistics for gas prices of transactions in the
+    /// mempool.
+    ///
+    /// It retrieves a specified number of transactions, sorted by descending
+    /// gas price, and calculates the average, maximum, minimum and median
+    /// prices. If `max_transactions` is not provided, defaults to all
+    /// transactions in the mempool. In the absence of transactions, will
+    /// default to a gas price of 1.
+    ///
+    /// # Arguments
+    /// * `max_transactions` - Optional maximum number of transactions to
+    ///   consider.
+    ///
+    /// # Returns
+    /// A JSON object encapsulating the statistics, or an error if processing
+    /// fails.
+    async fn get_gas_price(
+        &self,
+        max_transactions: Option<usize>,
+    ) -> anyhow::Result<ResponseData> {
+        let max_transactions = max_transactions.unwrap_or(usize::MAX);
```

Comment:
Wouldn't be better to get this out of the function and accept usize directly?

Reply:
Will do